pkg/kubelet/kuberuntime/helpers.go

```diff
@@ -62,7 +62,7 @@ type podSandboxByCreated []*runtimeapi.PodSandbox
 
 func (p podSandboxByCreated) Len() int           { return len(p) }
 func (p podSandboxByCreated) Swap(i, j int)      { p[i], p[j] = p[j], p[i] }
-func (p podSandboxByCreated) Less(i, j int) bool { return p[i].CreatedAt > p[j].CreatedAt }
```

Comment:
This is sorting from newest to oldest.  A newer podSandbox has a greater CreatedAt.  So the original implementation is correct.